From 59c73853fac05aa9579951d98a59c9c396faa1da Mon Sep 17 00:00:00 2001
From: Pratham Malaviya <prathamm@mechasystems.com>
Date: Tue, 27 Jan 2026 13:18:13 +0530
Subject: [PATCH 1/3] feat: support for touch to wake

Adds support for touch to wake for FTS touch
controller.

Signed-off-by: Pratham Malaviya <prathamm@mechasystems.com>

diff --git a/arch/arm64/boot/dts/freescale/imx8mp-mecha-comet-m-gen6.dts b/arch/arm64/boot/dts/freescale/imx8mp-mecha-comet-m-gen6.dts
index 56c18948eb82..cf0e0c12ec57 100644
--- a/arch/arm64/boot/dts/freescale/imx8mp-mecha-comet-m-gen6.dts
+++ b/arch/arm64/boot/dts/freescale/imx8mp-mecha-comet-m-gen6.dts
@@ -349,6 +349,7 @@ touchctl: focaltech@38 {
 		compatible = "focaltech,fts";
 		reg = <0x38>;
 		pinctrl-names = "default";
+		wakeup-source;
 		pinctrl-0 = <&pinctrl_touch_i2c>;
 		interrupt-parent = <&gpio4>;
 		interrupts = <5 IRQ_TYPE_EDGE_FALLING>;
-- 
2.25.1

